frontend/src/components/transactions/ActionMessage.tsx

```diff
@@ -48,25 +49,19 @@ const transactionMessageRenderers: TransactionMessageRenderers = {
   }: Props<T.DeployContract>) => <>{`Contract deployed: ${receiverId}`}</>,
   FunctionCall: ({
     transaction: { receiverId },
-    actionArgs
+    actionArgs,
+    showArg
   }: Props<T.FunctionCall>) => {
     const args = Buffer.from(actionArgs.args, "base64").toString();
-    let parameter;
-    if (args === "{}") {
-      parameter = "";
-    } else {
-      const argums = args.slice(1, args.length - 1).split(",");
-      parameter = argums.map(ar => {
-        const index = ar.indexOf(":");
-        return ar.slice(0, index);
-      });
-    }
     return (
       <>
-        {`Called method: ${
-          actionArgs.method_name
-        } with args {${parameter}} in contract: `}
+        {`Called method: '${actionArgs.method_name}' in contract: `}
         <AccountLink accountId={receiverId} />
+        {showArg ? (
+          <pre>
+            <i>Arguments in method: {args}</i>
```

Comment:
Let's move the caption "Arguments in method" outside of the `<pre>` block and make sure we have text wrapping (https://stackoverflow.com/questions/248011/how-do-i-wrap-text-in-a-pre-tag) for the code. I don't feel like we need italic here.